src/view/viewTabs.component.tsx

```diff
@@ -4,7 +4,8 @@ import React from 'react';
 import { Outlet, useLocation, useNavigate } from 'react-router-dom';
```

Comment:
There is still a `<Tabs<TabValue>` in here.

Reply:
remove it in the next pr